src/hotel/hotel.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { Hotel } from './hotel.model';
+
+@Injectable()
+export class HotelService {
+  private hotels: Hotel[] = [{
```

Comment:
It will be more cleaner if you move lines 6 - 15 to a fixture file.

Reply:
ok sir, I'll work on that